libs/ledger/include/ledger/chaincode/contract.hpp

```diff
@@ -291,5 +292,15 @@ StateAdapter::Status Contract::SetStateRecord(T const &record, ConstByteArray co
   return state().Write(std::string{key}, data.pointer(), data.size());
 }
 
+class ContractContextAttacher
```

Comment:
Could we have the class in contract_context_attacher.hpp? Optimally with a class-level comment on its utility

Reply:
Done.